### DISCUSSION TITLE
how to create a discussion ignore this is for a test purpose

### DISCUSSION THREAD

**Discussion by natnaelbelayneh** [community/community#110179] [New to GitHub]
this is a test

**Comment by ettaboyle** [comment]
👋 Welcome to the GitHub Community on Discussions, @natnae54! Happy to have you here 🙂

In our [Community Code of Conduct](https://github.com/community/community/blob/main/CODE_OF_CONDUCT.md), we ask that members contribute in a positive and constructive way. This type of post is considered "off topic" for our Community Discussions. Any repeat and future violations may result in a temporary or indefinite block from the Community. Thanks for understanding. 

Best of luck!
